core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -555,6 +570,17 @@ private case class OffsetsToClean(firstDirtyOffset: Long,
 }
 
 private[log] object LogCleanerManager extends Logging {
+  private val UncleanablePartitionsCountMetricName = "uncleanable-partitions-count"
+  private val UncleanableBytesMetricName = "uncleanable-bytes"
+  private val MaxDirtyPercentMetricName = "max-dirty-percent"
+  private val TimeSinceLastRunMsMetricName = "time-since-last-run-ms"
+
+  private[log] val GaugeMetricNameWithTag = new java.util.HashMap[String, java.util.List[java.util.Map[String, String]]]()
```

Comment:
Why variable names are not camelCase? 
 
@hudeqi @divijvaidya

Reply:
From https://kafka.apache.org/coding-guide:

`Constants should be camel case with an initial capital LikeThis not LIKE_THIS.`

Reply:
Let me take that back.

If it were a def function, we would probably have used camel case. For constants such as `TimeSinceLastRunMsMetricName` definitely first capital letter. But for variable names such as `GaugeMetricNameWithTag` we should probably use small case. 

Also, this `GaugeMetricNameWithTag` is an instance specific member and does not belong in companion object. @hudeqi would you please move this into the class itself, since it stores a state which is specific to the instance.

Reply:
Thanks! @divijvaidya @kamalcph. I will pay attention to this issue in subsequent PRs of this type. I have submitted the update of this PR to this minor PR(https://github.com/apache/kafka/pull/13958), thanks for the review.